### DISCUSSION TITLE
How to force light theme

### DISCUSSION THREAD

**Discussion by yashwanthkrishnab** [zitadel/zitadel#5799] [Q&A] [answered]
```zitadel/console/src/styles.scss```
```sass
$caos-dark-app-theme: (
  primary: mat-palette($caos-dark-primary),
  accent: mat-palette($caos-dark-primary),
  warn: mat-palette($caos-dark-warn),
  is-dark: true,
  foreground: $caos-dark-theme-foreground,
  background: $caos-dark-theme-background,
);

$caos-light-app-theme: (
  primary: mat-palette($caos-light-primary),
  accent: mat-palette($caos-light-primary),
  warn: mat-palette($caos-light-warn),
  is-dark: false,
  foreground: $caos-light-theme-foreground,
  background: $caos-light-theme-background,
);
```

By swapping these theme variables can we force light theme @hifabienne

**Comment by fforootd** [comment]
Thanks for raising this. Under what scenarios do you want to force the light-theme?

@peintnermax FYI

**Reply by yashwanthkrishnab** [reply]
thanks for the support. if my single page application doesn't support dark mode its atypical for login pages to have the feature @fforootd .
_found how to force light theme_

**Reply by yashwanthkrishnab** [reply, marked as answer]
```zitadel/internal/api/ui/login/static/resources/scripts/theme.js```
below change will force light theme
```js
const usesDarkTheme = false;
```

**Reply by fforootd** [reply]
Ah I see, that makes sense 😁
